src/style/bootstrap.css

```diff
@@ -4174,7 +4174,7 @@ input[type="button"].btn-block {
   display: inline-block;
   padding-top: 0.3125rem;
   padding-bottom: 0.3125rem;
-  margin-right: 1rem;
+  margin: 0 1rem;
```

Comment:
you should not change the Bootstrap prop, you can write your own class for this, no one knows what can go wrong if you change something in the basic things